Binary files pgmodeler.orig/crashhandler/crashhandler and pgmodeler/crashhandler/crashhandler differ
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/crashhandler/src/crashhandler.cpp pgmodeler/crashhandler/src/crashhandler.cpp
--- pgmodeler.orig/crashhandler/src/crashhandler.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/crashhandler/src/crashhandler.cpp	2013-06-13 18:58:21.108319624 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QDateTime>
 #include "crashhandler.h"
 
 const char CrashHandler::CHR_DELIMITER=static_cast<char>(3);
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/crashhandler/src/main.cpp pgmodeler/crashhandler/src/main.cpp
--- pgmodeler.orig/crashhandler/src/main.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/crashhandler/src/main.cpp	2013-06-13 18:55:55.174147795 +0400
@@ -19,6 +19,7 @@
 #include "crashhandler.h"
 #include <QApplication>
 #include <QTranslator>
+#include <QtGui/QStyleFactory>
 
 int main(int argc, char **argv)
 {
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/baseobjectview.cpp pgmodeler/libobjrenderer/src/baseobjectview.cpp
--- pgmodeler.orig/libobjrenderer/src/baseobjectview.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/baseobjectview.cpp	2013-06-13 18:33:47.506494955 +0400
@@ -17,6 +17,9 @@
 */
 
 #include "baseobjectview.h"
+#include <QtGui/QGraphicsItem>
+#include <QtGui/QGraphicsScene>
+#include <QtGui/QGraphicsSceneMouseEvent>
 
 map<QString, QTextCharFormat> BaseObjectView::font_config;
 map<QString, QColor *> BaseObjectView::color_config;
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/baseobjectview.h pgmodeler/libobjrenderer/src/baseobjectview.h
--- pgmodeler.orig/libobjrenderer/src/baseobjectview.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/baseobjectview.h	2013-06-13 18:33:47.507494969 +0400
@@ -25,7 +25,8 @@
 #ifndef BASE_OBJECT_VIEW_H
 #define BASE_OBJECT_VIEW_H
 
-#include <QtWidgets>
+#include <QtGui/QGraphicsItem>
+#include <QtGui/QTextCharFormat>
 #include "basegraphicobject.h"
 #include "baserelationship.h"
 #include "xmlparser.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/basetableview.cpp pgmodeler/libobjrenderer/src/basetableview.cpp
--- pgmodeler.orig/libobjrenderer/src/basetableview.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/basetableview.cpp	2013-06-13 18:33:47.507494969 +0400
@@ -17,6 +17,8 @@
 */
 
 #include "basetableview.h"
+#include <QtGui/QGraphicsScene>
+#include <QtGui/QGraphicsSceneMouseEvent>
 
 bool BaseTableView::hide_ext_attribs=false;
 
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/objectsscene.cpp pgmodeler/libobjrenderer/src/objectsscene.cpp
--- pgmodeler.orig/libobjrenderer/src/objectsscene.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/objectsscene.cpp	2013-06-13 18:33:47.507494969 +0400
@@ -16,6 +16,8 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QPainter>
+#include <QtGui/QGraphicsSceneMouseEvent>
 #include "objectsscene.h"
 
 bool ObjectsScene::align_objs_grid=false;
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/objectsscene.h pgmodeler/libobjrenderer/src/objectsscene.h
--- pgmodeler.orig/libobjrenderer/src/objectsscene.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/objectsscene.h	2013-06-13 18:33:47.508494984 +0400
@@ -25,8 +25,8 @@
 #ifndef OBJECTS_SCENE_H
 #define OBJECTS_SCENE_H
 
-#include <QtWidgets>
 #include <QPrinter>
+#include <QtGui/QGraphicsScene>
 #include "relationshipview.h"
 #include "graphicalview.h"
 #include "tableview.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/relationshipview.cpp pgmodeler/libobjrenderer/src/relationshipview.cpp
--- pgmodeler.orig/libobjrenderer/src/relationshipview.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/relationshipview.cpp	2013-06-13 18:33:47.508494984 +0400
@@ -17,6 +17,7 @@
 */
 
 #include "relationshipview.h"
+#include <QtGui/QGraphicsSceneMouseEvent>
 
 bool RelationshipView::hide_name_label=false;
 
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libobjrenderer/src/schemaview.cpp pgmodeler/libobjrenderer/src/schemaview.cpp
--- pgmodeler.orig/libobjrenderer/src/schemaview.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libobjrenderer/src/schemaview.cpp	2013-06-13 18:33:47.508494984 +0400
@@ -16,6 +16,8 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QGraphicsScene>
+#include <QtGui/QGraphicsSceneMouseEvent>
 #include "schemaview.h"
 
 SchemaView::SchemaView(Schema *schema) : BaseObjectView(schema)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/aggregatewidget.h pgmodeler/libpgmodeler_ui/src/aggregatewidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/aggregatewidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/aggregatewidget.h	2013-06-13 18:33:47.509494998 +0400
@@ -25,7 +25,6 @@
 #ifndef AGGREGATE_WIDGET_H
 #define AGGREGATE_WIDGET_H
 
-#include <QtWidgets>
 #include "baseobjectwidget.h"
 #include "ui_aggregatewidget.h"
 #include "pgsqltypewidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/appearanceconfigwidget.cpp pgmodeler/libpgmodeler_ui/src/appearanceconfigwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/appearanceconfigwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/appearanceconfigwidget.cpp	2013-06-13 18:33:47.509494998 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QGraphicsView>
 #include "appearanceconfigwidget.h"
 
 AppearanceConfigWidget::AppearanceConfigWidget(QWidget * parent) : QWidget(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/appearanceconfigwidget.h pgmodeler/libpgmodeler_ui/src/appearanceconfigwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/appearanceconfigwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/appearanceconfigwidget.h	2013-06-13 18:33:47.509494998 +0400
@@ -25,6 +25,7 @@
 #ifndef APPEARANCE_CONFIG_WIDGET_H
 #define APPEARANCE_CONFIG_WIDGET_H
 
+#include <QtGui/QColorDialog>
 #include "ui_appearanceconfigwidget.h"
 #include "baseconfigwidget.h"
 #include "objectsscene.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/baseform.h pgmodeler/libpgmodeler_ui/src/baseform.h
--- pgmodeler.orig/libpgmodeler_ui/src/baseform.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/baseform.h	2013-06-13 18:33:47.510495013 +0400
@@ -25,7 +25,6 @@
 #ifndef BASE_FORM_H
 #define BASE_FORM_H
 
-#include <QtWidgets>
 #include "ui_baseform.h"
 #include "exception.h"
 #include "messagebox.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/baseobjectwidget.cpp pgmodeler/libpgmodeler_ui/src/baseobjectwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/baseobjectwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/baseobjectwidget.cpp	2013-06-13 18:33:47.510495013 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QKeyEvent>
 #include "baseobjectwidget.h"
 #include "permissionwidget.h"
 
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/baseobjectwidget.h pgmodeler/libpgmodeler_ui/src/baseobjectwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/baseobjectwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/baseobjectwidget.h	2013-06-13 18:33:47.510495013 +0400
@@ -25,7 +25,6 @@
 #ifndef BASE_OBJECT_WIDGET_H
 #define BASE_OBJECT_WIDGET_H
 
-#include <QtWidgets>
 #include "databasemodel.h"
 #include "operationlist.h"
 #include "baseform.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/elementswidget.h pgmodeler/libpgmodeler_ui/src/elementswidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/elementswidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/elementswidget.h	2013-06-13 18:33:47.510495013 +0400
@@ -25,7 +25,6 @@
 #ifndef ELEMENT_WIDGET_H
 #define ELEMENT_WIDGET_H
 
-#include <QtWidgets>
 #include "ui_elementswidget.h"
 #include "objecttablewidget.h"
 #include "objectselectorwidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/generalconfigwidget.cpp pgmodeler/libpgmodeler_ui/src/generalconfigwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/generalconfigwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/generalconfigwidget.cpp	2013-06-13 18:33:47.511495027 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QProcess>
 #include "generalconfigwidget.h"
 
 GeneralConfigWidget::GeneralConfigWidget(QWidget * parent) : QWidget(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/mainwindow.cpp pgmodeler/libpgmodeler_ui/src/mainwindow.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/mainwindow.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/mainwindow.cpp	2013-06-13 18:33:47.511495027 +0400
@@ -16,6 +16,9 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QFileDialog>
+#include <QtCore/QTimer>
+#include <QtGui/QDesktopServices>
 #include "mainwindow.h"
 #include "textboxwidget.h"
 #include "sourcecodewidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/mainwindow.h pgmodeler/libpgmodeler_ui/src/mainwindow.h
--- pgmodeler.orig/libpgmodeler_ui/src/mainwindow.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/mainwindow.h	2013-06-13 18:33:47.511495027 +0400
@@ -25,8 +25,8 @@
 #ifndef MAIN_WINDOW_H
 #define MAIN_WINDOW_H
 
-#include <QtWidgets>
 #include <QPrintDialog>
+#include <QtCore/QTimer>
 #include "ui_mainwindow.h"
 #include "modelwidget.h"
 #include "aboutform.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelexportform.cpp pgmodeler/libpgmodeler_ui/src/modelexportform.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/modelexportform.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelexportform.cpp	2013-06-13 18:33:47.511495027 +0400
@@ -16,6 +16,8 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QTimer>
+#include <QtGui/QFileDialog>
 #include "modelexportform.h"
 #include "taskprogresswidget.h"
 #include "configurationform.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelexporthelper.cpp pgmodeler/libpgmodeler_ui/src/modelexporthelper.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/modelexporthelper.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelexporthelper.cpp	2013-06-13 18:33:47.512495041 +0400
@@ -1,3 +1,4 @@
+#include <QtGui/QPainter>
 #include "modelexporthelper.h"
 
 ModelExportHelper::ModelExportHelper(QObject *parent) : QObject(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelobjectswidget.cpp pgmodeler/libpgmodeler_ui/src/modelobjectswidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/modelobjectswidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelobjectswidget.cpp	2013-06-13 18:33:47.512495041 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QDesktopWidget>
 #include "modelobjectswidget.h"
 
 ModelObjectsWidget::ModelObjectsWidget(bool simplified_view, QWidget *parent) : QWidget(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelobjectswidget.h pgmodeler/libpgmodeler_ui/src/modelobjectswidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/modelobjectswidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelobjectswidget.h	2013-06-13 18:33:47.512495041 +0400
@@ -25,7 +25,7 @@
 #ifndef MODEL_OBJECTS_WIDGET_H
 #define MODEL_OBJECTS_WIDGET_H
 
-#include <QtWidgets>
+#include <QtCore/QSettings>
 #include "ui_modelobjectswidget.h"
 #include "modelwidget.h"
 #include "messagebox.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modeloverviewwidget.cpp pgmodeler/libpgmodeler_ui/src/modeloverviewwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/modeloverviewwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modeloverviewwidget.cpp	2013-06-13 18:33:47.512495041 +0400
@@ -16,6 +16,9 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QGraphicsView>
+#include <QtGui/QScrollBar>
+#include <QtGui/QMouseEvent>
 #include "modeloverviewwidget.h"
 #include "modelwidget.h"
 
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modeloverviewwidget.h pgmodeler/libpgmodeler_ui/src/modeloverviewwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/modeloverviewwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modeloverviewwidget.h	2013-06-13 18:33:47.513495056 +0400
@@ -27,7 +27,6 @@
 #ifndef MODEL_OVERVIEW_WIDGET_H
 #define MODEL_OVERVIEW_WIDGET_H
 
-#include <QtWidgets>
 #include "exception.h"
 #include "ui_modeloverviewwidget.h"
 #include "modelwidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelrestorationform.cpp pgmodeler/libpgmodeler_ui/src/modelrestorationform.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/modelrestorationform.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelrestorationform.cpp	2013-06-13 18:33:47.513495056 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QDateTime>
 #include "modelrestorationform.h"
 
 ModelRestorationForm::ModelRestorationForm(QWidget *parent, Qt::WindowFlags f) : QDialog(parent, f)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelrestorationform.h pgmodeler/libpgmodeler_ui/src/modelrestorationform.h
--- pgmodeler.orig/libpgmodeler_ui/src/modelrestorationform.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelrestorationform.h	2013-06-13 18:33:47.513495056 +0400
@@ -25,7 +25,6 @@
 #ifndef MODEL_RESTORATION_FORM_H
 #define MODEL_RESTORATION_FORM_H
 
-#include <QtWidgets>
 #include "globalattributes.h"
 #include "ui_modelrestorationform.h"
 
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelwidget.cpp pgmodeler/libpgmodeler_ui/src/modelwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/modelwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelwidget.cpp	2013-06-13 18:33:47.514495070 +0400
@@ -16,6 +16,10 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QTemporaryFile>
+#include <QtGui/QGraphicsView>
+#include <QtGui/QMenu>
+#include <QtGui/QScrollBar>
 #include "modelwidget.h"
 #include "sourcecodewidget.h"
 #include "databasewidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/modelwidget.h pgmodeler/libpgmodeler_ui/src/modelwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/modelwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/modelwidget.h	2013-06-13 18:33:47.514495070 +0400
@@ -25,7 +25,7 @@
 #ifndef MODEL_WIDGET_H
 #define MODEL_WIDGET_H
 
-#include <QtWidgets>
+#include <QtGui/QMenu>
 #include "databasemodel.h"
 #include "operationlist.h"
 #include "messagebox.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/objectfinderwidget.cpp pgmodeler/libpgmodeler_ui/src/objectfinderwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/objectfinderwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/objectfinderwidget.cpp	2013-06-13 18:33:47.514495070 +0400
@@ -16,6 +16,8 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/qevent.h>
+#include <QtGui/QGraphicsView>
 #include "objectfinderwidget.h"
 
 ObjectFinderWidget::ObjectFinderWidget(QWidget *parent) : QWidget(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/objectrenamewidget.h pgmodeler/libpgmodeler_ui/src/objectrenamewidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/objectrenamewidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/objectrenamewidget.h	2013-06-13 18:33:47.515495085 +0400
@@ -25,7 +25,6 @@
 #ifndef OBJECT_RENAME_WIDGET_H
 #define OBJECT_RENAME_WIDGET_H
 
-#include <QtWidgets>
 #include "ui_objectrenamewidget.h"
 #include "databasemodel.h"
 #include "operationlist.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/objectselectorwidget.h pgmodeler/libpgmodeler_ui/src/objectselectorwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/objectselectorwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/objectselectorwidget.h	2013-06-13 18:33:47.515495085 +0400
@@ -25,7 +25,6 @@
 #ifndef OBJECT_SELECTOR_WIDGET_H
 #define OBJECT_SELECTOR_WIDGET_H
 
-#include <QtWidgets>
 #include "databasemodel.h"
 #include "syntaxhighlighter.h"
 #include "modelobjectswidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/operationlistwidget.h pgmodeler/libpgmodeler_ui/src/operationlistwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/operationlistwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/operationlistwidget.h	2013-06-13 18:33:47.515495085 +0400
@@ -25,7 +25,6 @@
 #ifndef OPERATION_LIST_WIDGET_H
 #define OPERATION_LIST_WIDGET_H
 
-#include <QtWidgets>
 #include "ui_operationlistwidget.h"
 #include "modelwidget.h"
 #include "messagebox.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/pgsqltypewidget.h pgmodeler/libpgmodeler_ui/src/pgsqltypewidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/pgsqltypewidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/pgsqltypewidget.h	2013-06-13 18:33:47.515495085 +0400
@@ -25,7 +25,6 @@
 #ifndef PGSQL_TYPE_WIDGET_H
 #define PGSQL_TYPE_WIDGET_H
 
-#include <QtWidgets>
 #include "ui_pgsqltypewidget.h"
 #include "pgsqltypes.h"
 #include "syntaxhighlighter.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/pluginsconfigwidget.cpp pgmodeler/libpgmodeler_ui/src/pluginsconfigwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/pluginsconfigwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/pluginsconfigwidget.cpp	2013-06-13 18:33:47.515495085 +0400
@@ -16,6 +16,9 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QDesktopServices>
+#include <QtCore/QLibrary>
+#include <QtCore/QPluginLoader>
 #include "pluginsconfigwidget.h"
 
 PluginsConfigWidget::PluginsConfigWidget(QWidget *parent) : QWidget(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/pluginsconfigwidget.h pgmodeler/libpgmodeler_ui/src/pluginsconfigwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/pluginsconfigwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/pluginsconfigwidget.h	2013-06-13 18:33:47.516495099 +0400
@@ -25,6 +25,7 @@
 #ifndef PLUGINS_CONFIG_WIDGET_H
 #define PLUGINS_CONFIG_WIDGET_H
 
+#include <QtGui/QToolBar>
 #include "baseconfigwidget.h"
 #include "ui_pluginsconfigwidget.h"
 #include "objecttablewidget.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/schemawidget.cpp pgmodeler/libpgmodeler_ui/src/schemawidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/schemawidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/schemawidget.cpp	2013-06-13 18:33:47.516495099 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QColorDialog>
 #include "schemawidget.h"
 
 SchemaWidget::SchemaWidget(QWidget *parent): BaseObjectWidget(parent, OBJ_SCHEMA)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/syntaxhighlighter.cpp pgmodeler/libpgmodeler_ui/src/syntaxhighlighter.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/syntaxhighlighter.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/syntaxhighlighter.cpp	2013-06-13 18:33:47.516495099 +0400
@@ -16,6 +16,8 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QTextEdit>
+#include <QtGui/qevent.h>
 #include "syntaxhighlighter.h"
 
 SyntaxHighlighter::SyntaxHighlighter(QTextDocument *parent, bool auto_rehighlight) : QSyntaxHighlighter(parent)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/syntaxhighlighter.h pgmodeler/libpgmodeler_ui/src/syntaxhighlighter.h
--- pgmodeler.orig/libpgmodeler_ui/src/syntaxhighlighter.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/syntaxhighlighter.h	2013-06-13 18:33:47.516495099 +0400
@@ -25,7 +25,7 @@
 #ifndef SYNTAX_HIGHLIGHTER_H
 #define SYNTAX_HIGHLIGHTER_H
 
-#include <QtWidgets>
+#include <QtGui/QSyntaxHighlighter>
 #include <map>
 #include <vector>
 #include "exception.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/taskprogresswidget.cpp pgmodeler/libpgmodeler_ui/src/taskprogresswidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/taskprogresswidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/taskprogresswidget.cpp	2013-06-13 18:33:47.516495099 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QTimer>
 #include "taskprogresswidget.h"
 
 TaskProgressWidget::TaskProgressWidget(QWidget *parent, Qt::WindowFlags f) : QDialog(parent, f)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/taskprogresswidget.h pgmodeler/libpgmodeler_ui/src/taskprogresswidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/taskprogresswidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/taskprogresswidget.h	2013-06-13 18:33:47.517495114 +0400
@@ -25,7 +25,6 @@
 #ifndef TASK_PROGRESS_WIDGET_H
 #define TASK_PROGRESS_WIDGET_H
 
-#include <QtWidgets>
 #include "ui_taskprogresswidget.h"
 #include <map>
 #include "utf8string.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/textboxwidget.cpp pgmodeler/libpgmodeler_ui/src/textboxwidget.cpp
--- pgmodeler.orig/libpgmodeler_ui/src/textboxwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/textboxwidget.cpp	2013-06-13 18:33:47.517495114 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtGui/QColorDialog>
 #include "textboxwidget.h"
 
 TextboxWidget::TextboxWidget(QWidget *parent): BaseObjectWidget(parent, OBJ_TEXTBOX)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/libpgmodeler_ui/src/viewwidget.h pgmodeler/libpgmodeler_ui/src/viewwidget.h
--- pgmodeler.orig/libpgmodeler_ui/src/viewwidget.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/libpgmodeler_ui/src/viewwidget.h	2013-06-13 18:33:47.517495114 +0400
@@ -25,7 +25,6 @@
 #ifndef VIEW_WIDGET_H
 #define VIEW_WIDGET_H
 
-#include <QtWidgets>
 #include "baseobjectwidget.h"
 #include "ui_viewwidget.h"
 #include "objecttablewidget.h"
Binary files pgmodeler.orig/main/pgmodeler and pgmodeler/main/pgmodeler differ
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/main/src/main.cpp pgmodeler/main/src/main.cpp
--- pgmodeler.orig/main/src/main.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/main/src/main.cpp	2013-06-13 18:33:47.517495114 +0400
@@ -19,6 +19,10 @@
 #include "mainwindow.h"
 #include "application.h"
 #include <QTranslator>
+#include <QtCore/QDateTime>
+#include <QtGui/QStyleFactory>
+#include <QtGui/QSplashScreen>
+#include <QtGui/QBitmap>
 
 #ifndef Q_OS_WIN
 #include "execinfo.h"
Binary files pgmodeler.orig/main-cli/pgmodeler-cli and pgmodeler/main-cli/pgmodeler-cli differ
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/plugins/dummy/src/dummy.cpp pgmodeler/plugins/dummy/src/dummy.cpp
--- pgmodeler.orig/plugins/dummy/src/dummy.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/plugins/dummy/src/dummy.cpp	2013-06-13 18:33:47.518495128 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QTranslator>
 #include "dummy.h"
 #include "exception.h"
 #include "messagebox.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/plugins/dummy/src/dummy.h pgmodeler/plugins/dummy/src/dummy.h
--- pgmodeler.orig/plugins/dummy/src/dummy.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/plugins/dummy/src/dummy.h	2013-06-13 18:33:47.518495128 +0400
@@ -25,13 +25,15 @@
 #ifndef DUMMY_H
 #define DUMMY_H
 
+#include <QtCore/QtPlugin>
 #include "pgmodelerplugin.h"
 
 class Dummy: public QObject, public PgModelerPlugin {
 	private:
 		Q_OBJECT
 
-		Q_PLUGIN_METADATA(IID "br.com.pgmodeler.PgModelerPlugin" FILE "dummy.json")
+//?		Q_PLUGIN_METADATA(IID "br.com.pgmodeler.PgModelerPlugin" FILE "dummy.json")
+		Q_EXPORT_STATIC_PLUGIN2("br.com.pgmodeler.PgModelerPlugin", "dummy.json")
 
 		//! \brief Declares the interface which is used to implement the plugin
 		Q_INTERFACES(PgModelerPlugin)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/plugins/xml2object/src/xml2object.cpp pgmodeler/plugins/xml2object/src/xml2object.cpp
--- pgmodeler.orig/plugins/xml2object/src/xml2object.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/plugins/xml2object/src/xml2object.cpp	2013-06-13 18:33:47.518495128 +0400
@@ -16,6 +16,7 @@
 # Also, you can get the complete GNU General Public License at <http://www.gnu.org/licenses/>
 */
 
+#include <QtCore/QTranslator>
 #include "xml2object.h"
 #include "exception.h"
 #include "messagebox.h"
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/plugins/xml2object/src/xml2object.h pgmodeler/plugins/xml2object/src/xml2object.h
--- pgmodeler.orig/plugins/xml2object/src/xml2object.h	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/plugins/xml2object/src/xml2object.h	2013-06-13 18:33:47.518495128 +0400
@@ -25,6 +25,7 @@
 #ifndef XML2OBJECT_H
 #define XML2OBJECT_H
 
+#include <QtCore/qplugin.h>
 #include "pgmodelerplugin.h"
 #include "xml2objectwidget.h"
 
@@ -32,7 +33,8 @@
 	private:
 		Q_OBJECT
 
-		Q_PLUGIN_METADATA(IID "br.com.pgmodeler.PgModelerPlugin" FILE "xml2object.json")
+//??		Q_PLUGIN_METADATA(IID "br.com.pgmodeler.PgModelerPlugin" FILE "xml2object.json")
+		Q_EXPORT_STATIC_PLUGIN2("br.com.pgmodeler.PgModelerPlugin", "xml2object.json")
 
 		//! \brief Declares the interface which is used to implement the plugin
 		Q_INTERFACES(PgModelerPlugin)
diff -NurEbBH -x '*.so' -x '*.so.*' -x '*.o' -x Makefile -x '*QTutf8' -x '*nbproject*' '-x*.QT4' -x '*.bak' '-xmoc_*' -x 'qrc_*' -x 'ui_*' -x '*.sed' pgmodeler.orig/plugins/xml2object/src/xml2objectwidget.cpp pgmodeler/plugins/xml2object/src/xml2objectwidget.cpp
--- pgmodeler.orig/plugins/xml2object/src/xml2objectwidget.cpp	2013-06-12 22:30:51.000000000 +0400
+++ pgmodeler/plugins/xml2object/src/xml2objectwidget.cpp	2013-06-13 18:33:47.518495128 +0400
@@ -1,3 +1,4 @@
+#include <QtGui/QFileDialog>
 #include "xml2objectwidget.h"
 
 Xml2ObjectWidget::Xml2ObjectWidget(QWidget *parent, Qt::WindowFlags f) : QDialog(parent,f)
